x-pack/plugins/ml/server/models/data_recognizer/modules/siem_winlogbeat_auth/ml/datafeed_windows_rare_user_type10_remote_login.json

```diff
@@ -7,9 +7,40 @@
   "query": {
     "bool": {
       "filter": [
-        {"term": {"event.type": "authentication_success"}},
-        {"term": {"winlog.event_data.LogonType": "10"}},
-        {"term": {"agent.type": "winlogbeat"}}
+        {
+          "term": {
+            "winlog.event_data.LogonType": "10"
+          }
+        },
+        {
+          "term": {
+            "event.code": "4624"
+          }
+        }
+      ],
+      "must": [
+        {
+          "bool": {
+            "should": [
+              {
+                "match": {
```

Comment:
I believe this should be `term` instead of `match` per discussions w/ StackML team

Reply:
OK these are similar to the OR tests in the DF queries for the 7.11 jobs - I thought we needed to write the OR clauses like this?

Reply:
It is in reference to: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html